src/app/components/templates/Navbar/Navbar.tsx

```diff
@@ -27,11 +30,25 @@ const Navbar = async ({ lng = fallbackLng, t }: { lng?: Locale; t: T }) => {
   return (
     <header className="bg-primary px-4 py-2 md:py-0.5 text-white">
       <div className="flex justify-between items-center">
-        <Analytics event={LogEvents.navBarClickLogo}>
-          <Link href={`/${lng}`}>
-            <ZiggleLogo className="w-20 md:w-auto" />
+        <div className="flex items-center gap-5">
+          <Analytics event={LogEvents.navBarClickLogo}>
+            <Link href={`/${lng}`}>
+              {lng === 'en' ? (
+                <ZiggleEnglishLogo className="h-15 md:w-auto" />
+              ) : (
+                <ZiggleKoreanLogo className="h-15 md:w-auto" />
+              )}
```

Comment:
기본 언어가 english이고 korean이 특수한 형태로 가는게 좋을 것 같아요
```suggestion
              {lng === 'ko' ? (
              	<ZiggleKoreanLogo className="h-15 md:w-auto" />
              ) : (
                <ZiggleEnglishLogo className="h-15 md:w-auto" />
              )}
```

Reply:
피드백 반영했습니다!